### ISSUE TITLE
Not able to create onnx model from tensorflow after loading the bento.

### ISSUE THREAD

**Issue by arvindhbti** [bentoml/BentoML#2111]
Describe the problem:
Currently I am working with creating the bento from tensorflow and onnx models.

So, I am creating models with two inputs x1,x2. when I custom train the model and create the bento just after packing the bento with onnx, tensorflow models (without saving) then its working fine.
But ,
when I save the bento after packing with tensorflow/onnx models and then load it to custom train the models. During testing, tensorflow bento is working fine but when I predict with onnx then it is showing error as :

```
     39             model = classifier.artifacts.classifier_onnx
     40             input_name2 = model.get_inputs()[0].name
---> 41             input_name1 = model.get_inputs()[1].name
     42             pred_onnx = model.run(None, {input_name2 : image,input_name1:vec})[0]
     43 

IndexError: list index out of range
```

It means it is taking only one input but it should take two inputs as expected in onnx model.
And, its working fine when I try to create the bento just after packing with tensorflow/onnx models.

Any suggestions?

**Comment by parano** [comment]
Hi @arvindhbti - could you share your service definition code to help us investigate further?

Also BentoML 1.0 preview release has just arrived where we improved support for Tensorflow and ONNX. In 1.0, we unified the model inference API using a Runner abstraction to avoid confusion scenarios like this where model object before and after save may expose slightly different interface for the user.

**Comment by ssheng** [comment]
BentoML has released official 1.0.0. Could you please give it a try? Let us know if this problem persists.
